mempool/src/shared_mempool.rs

```diff
@@ -31,28 +42,40 @@ use tokio::{
 };
 use vm_validator::vm_validator::{get_account_state, TransactionValidation};
 
+const WORKER_CHANNEL_BUFFER_SIZE: usize = 5;
```

Comment:
what's a reason to have it set to 5? How smp will behave if network keeps blipping and worker can't process messages in time?